zebra/zebra_nhg.c

```diff
@@ -307,6 +307,25 @@ static int zebra_nhg_insert_id(struct nhg_hash_entry *nhe)
 	return 0;
 }
 
+struct nhg_hash_entry *zebra_presvd_nhg_lookup(struct nhg_hash_entry *lookup)
+{
+	if (lookup == NULL)
+		return lookup;
```

Comment:
hash_lookup() returns NULL itself, no need to check here. We can just use `return hash_lookup(...)`.